backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidZipFileHandle.java

```diff
@@ -84,7 +84,7 @@ public FileHandle child(String name) {
 	public FileHandle sibling(String name) {
 		if (file.getPath().length() == 0)
 			throw new GdxRuntimeException("Cannot get the sibling of the root.");
-		return new AndroidZipFileHandle(new File(file.getParent(), name), type);
+		return Gdx.files.internal(file.getParent() + "/" + name); //this way we can find the sibling even if it's not inside the obb
```

Comment:
Type should always be Internal in case of AndroidZipFiles, but can do if you think.